src/test/java/com/flightstats/hub/webhook/WebhookCoordinatorTest.java

```diff
@@ -106,7 +112,11 @@ void testWhenNewWebhook_getsAddedToServerManagingFewestWebhooks() {
 
     @Test
     void testWhenInactiveWebhook_getsAddedToServerManagingFewestWebhooks() {
-        when(webhookLeaderState.isActiveWebhook(WEBHOOK_NAME)).thenReturn(false);
+        when(webhookLeaderState.getState(WEBHOOK_NAME)).thenReturn(
+                WebhookLeaderState.RunningState.builder()
+                        .leadershipAcquired(false)
+                        .runningServers(newHashSet())
+                        .build());
 
```

Comment:
Code will be easier to read if this "WebhookLeaderState.RunningState.builder()
                        .leadershipAcquired(false)
                        .runningServers(newHashSet())
                        .build()"
is pulled in to separate method. It is redundant.